[2016-03-21 00:03:32] local.ERROR: exception 'Symfony\Component\Console\Exception\CommandNotFoundException' with message 'Command "inspire" is not defined.' in /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php:512
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(183): Symfony\Component\Console\Application->find('inspire')
#1 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /Applications/XAMPP/xamppfiles/htdocs/untitled/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-03-21 00:04:19] local.ERROR: exception 'Symfony\Component\Console\Exception\CommandNotFoundException' with message 'Command "inspire" is not defined.' in /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php:512
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(183): Symfony\Component\Console\Application->find('inspire')
#1 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /Applications/XAMPP/xamppfiles/htdocs/untitled/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-03-21 00:26:52] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1045] Access denied for user 'deyangzhao'@'localhost' (using password: YES)' in /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:55
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(55): PDO->__construct('mysql:host=127....', 'deyangzhao', 'zdy931023@', Array)
#1 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connection.php(864): call_user_func(Object(Closure))
#5 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Connection.php(342): Illuminate\Database\Connection->run('select * from i...', Array, Object(Closure))
#8 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Schema/MySqlBuilder.php(21): Illuminate\Database\Connection->select('select * from i...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Migrations/DatabaseMigrationRepository.php(141): Illuminate\Database\Schema\MySqlBuilder->hasTable('migrations')
#10 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php(405): Illuminate\Database\Migrations\DatabaseMigrationRepository->repositoryExists()
#11 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Console/Migrations/MigrateCommand.php(103): Illuminate\Database\Migrations\Migrator->repositoryExists()
#12 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Database/Console/Migrations/MigrateCommand.php(58): Illuminate\Database\Console\Migrations\MigrateCommand->prepareDatabase()
#13 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#14 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#15 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Console/Command.php(169): Illuminate\Container\Container->call(Array)
#16 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Console/Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(791): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /Applications/XAMPP/xamppfiles/htdocs/untitled/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 /Applications/XAMPP/xamppfiles/htdocs/untitled/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 {main}  
[2016-03-24 02:26:16] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Barryvdh\Debugbar\ServiceProvider' not found' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-03-24 02:27:11] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Barryvdh\Debugbar\ServiceProvider' not found' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-03-30 22:20:10] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'CreatePostsTable' not found' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php:336
Stack trace:
#0 {main}  
[2016-03-30 22:46:03] local.ERROR: exception 'Symfony\Component\Console\Exception\CommandNotFoundException' with message 'Command "m" is ambiguous (make:seed, make:migration:schema and 21 more).' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/symfony/console/Application.php:529
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/symfony/console/Application.php(183): Symfony\Component\Console\Application->find('m')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-03-30 22:56:05] local.ERROR: exception 'Symfony\Component\Console\Exception\CommandNotFoundException' with message 'Command "namespace" is not defined.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/symfony/console/Application.php:512
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/symfony/console/Application.php(183): Symfony\Component\Console\Application->find('namespace')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-03-31 00:10:08] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(123): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#28 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#31 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#33 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#34 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#35 {main}  
[2016-03-31 00:10:08] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(152): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2016-03-31 00:10:42] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(123): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#28 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#31 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#33 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#34 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#35 {main}  
[2016-03-31 00:10:42] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(152): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2016-03-31 00:10:46] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(123): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#28 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#31 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#33 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#34 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#35 {main}  
[2016-03-31 00:10:46] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(152): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2016-03-31 00:10:57] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(123): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#28 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#31 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#32 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#33 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#34 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#35 {main}  
[2016-03-31 00:10:57] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Container/Container.php(633): Illuminate\Container\Container->build('ProgramPlanner\\...', Array)
#8 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('ProgramPlanner\\...', Array)
#9 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(152): Illuminate\Foundation\Application->make('ProgramPlanner\\...')
#10 /Applications/XAMPP/xamppfiles/htdocs/program-planner/laravel/public/index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
